modules/ROOT/pages/testing-writing-your-first-munit-test.adoc

```diff
@@ -0,0 +1,201 @@
+= Creating Your Test 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To test your module, you must create MUnit tests because MTF is built on top of MUnit. 
+
+This section provides a step-by-step example for testing a module using MUnit. This example is based on the module project generated with the Mule Extensions Archetype in xref:getting-started.adoc#generating-a-project-using-the-maven-archetype-directly[Creating Your First SDK Project].
```

Comment:
Avoid self-referential language like "this section".

Reply:
You could combine the two sentences into something like "This step-by-step example, based on..."